src/Orders/WebhookCallOrderHistory.php

```diff
@@ -0,0 +1,110 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Craftzing\Laravel\MollieWebhooks\Orders;
+
+use Craftzing\Laravel\MollieWebhooks\Queries\LatestMollieWebhookCallByResourceId;
+use Craftzing\Laravel\MollieWebhooks\Refunds\RefundId;
+use Craftzing\Laravel\MollieWebhooks\WebhookPayloadFragment;
+use Mollie\Api\Types\RefundStatus;
+use Spatie\WebhookClient\Models\WebhookCall;
+
+use function array_merge;
+use function compact;
+
+final class WebhookCallOrderHistory implements OrderHistory
```

Comment:
It's a small detail, but make sure to replace the `payment` reference with `order` in all comments as well.